src/qt/bdapaddlinkdialog.cpp

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) 2016-2019 Duality Blockchain Solutions Developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "bdapaddlinkdialog.h"
+#include "bdapfeespopup.h"
```

Comment:
add a space for header file

Reply:
Thanks, see https://github.com/duality-solutions/Dynamic/pull/276/commits/a0da4d622f26750a6df49e7acc59768095eb228d